### DISCUSSION TITLE
It's... well not alive yet. But It's something darn it!

### DISCUSSION THREAD

**Discussion by sjohnson1021** [AxionDrak/GameCube-Backup-Manager#88] [Show and tell]
Thoughts on this so far? Minimal effort put in, mainly just a dev playground.. but..?
<img width="1634" alt="image" src="https://user-images.githubusercontent.com/8071133/188059665-0307c630-717a-4c65-a1a5-b5764e282ca6.png">

I'd very much like more of a grid system,,

**Comment by sjohnson1021** [comment]
So for local (like where the app is running) file access. ... There... .. Are going to be some hoops

**Comment by laetemn** [comment]
> Thoughts on this so far? Minimal effort put in, mainly just a dev playground.. but..? <img alt="image" width="1634" src="https://user-images.githubusercontent.com/8071133/188059665-0307c630-717a-4c65-a1a5-b5764e282ca6.png">
> 
> I'd very much like more of a grid system,,

This is so beautiful!!! 😍
